core/examples/luigi-sample-angular/e2e/tests/navigation.spec.js

```diff
@@ -115,5 +115,10 @@ describe('Navigation', () => {
         expect(loc.hash).to.eq('#/projects/pr2/dps/dps1');
       });
     });
+    it('should hide left Nav', () => {
+      cy.visit('/#/projects/pr1/navCollapse');
+      cy.get('.no-left-nav');
```

Comment:
It does. It checks if there is any element with `no-left-nav` class. It's important because this class moves our _iframeContainer_ to the left edge. If you change the classname to anything else, Cypress won't be able to find it and the test will fail:
![image](https://user-images.githubusercontent.com/22825433/49211112-19166000-f3bf-11e8-845d-93cf1b81b250.png)